src/testtool.c

```diff
@@ -1196,7 +1196,6 @@ void all_stack_dump(DLword start, DLword end, DLword silent)
           print(get_framename((struct frameex1 *)stkptr));
           printf(" [");
           if (((FX *)stkptr)->fast) printf("fast,");
-          if (((FX *)stkptr)->native) printf("native,");
```

Comment:
This one should probably stay until we remove the "native" flag bit from the Frame Extension, likewise for the Function Header struct which also has a "native" flag.

Reply:
I removed them in `inc/stack.h` and marked what they used to be. Or are you talking about Lisp side? But C side, everything is gone and a few foot prints left as a trail to the past.

Reply:
Looks as though I got started in the middle somewhere, so I'd missed that you'd dummied out the "native" flag bit.
If all the traces (bar history pointers) are gone, then this is fine.  I/we do need to look at the Lisp side to make sure there are no dangling references into the emulator code.

Reply:
Looks as though I got started in the middle somewhere, so I'd missed that you'd dummied out the "native" flag bit.
If all the traces (bar history pointers) are gone, then this is fine.  I/we do need to look at the Lisp side to make sure there are no dangling references into the emulator code.